js/platform.ts

```diff
@@ -0,0 +1,3 @@
+// Dummy. Injected in rollup.config.js
+export const arch = "";
```

Comment:
maybe something other than empty string? Like `"unknown"`?

Reply:
Also, just thinking out loud, is that when these types get extracted and bundled up, because this is a `const`, TypeScript I suspect will create a literal type of `""` for these, which isn't the best.  What would be better would be to have a literal union type in here so that when it gets bundled into the types, end users have a valid type they can use, like:

```ts
export const arch: "x32'" | "x64" | "unknown" = "unknown";
export const platform: "darwin" | "linux" | "win32" | "unknown" = "unknown";
```

Reply:
Yeah I did notice this behavior when trying to add basic tests. I can definitely add the union type, just need to update the type if node will be supported on other new platforms (fuschia?)